soc/v2/raspberrypi/rp2/Kconfig.series

```diff
@@ -17,3 +17,9 @@ config SOC_SERIES_RP2XXX
 	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
 	help
 	  Enable support for Raspberry Pi RP2 MCU series
+
+config SOC_SERIES
+	default "rp2" if SOC_SERIES_RP2XXX
+
+config SOC
+	default "rp2040" if SOC_SERIES_RP2XXX
```

Comment:
should change if following the other comment related to `Kconfig.soc` and `Kconfig.series` files.